packages/@okta/vuepress-site/docs/reference/api/user-types/index.md

```diff
@@ -0,0 +1,432 @@
+    "schema": {
+      "href": "https://{yourOktaDomain}/api/v1/meta/schemas/user/oscfnly5cQjJT9PnR0g4",
+      "method": "GET",
+      "rel": "schema"
+    },
+    "self": {
+      "href": "https://{yourOktaDomain}/api/v1/meta/types/user/otyfnly5cQjJT9PnR0g4",
+      "method": "GET",
+      "rel": "self"
+    }
+  },
+  "created": "2019-04-10T02:00:01.000Z",
+  "createdBy": "00ufnlhzppWItClAI0g4",
+  "default": false,
+  "description": "Any description that means something useful to you",
+  "displayName": "Display Name for UI",
+  "id": "otyfnly5cQjJT9PnR0g4",
+  "lastUpdated": "2019-04-10T02:00:01.000Z",
+  "lastUpdatedBy": "00ufnlhzppWItClAI0g4",
+  "name": "aNewType",
+  "ref": {
+    "id": "otyfnly5cQjJT9PnR0g4",
+    "refClass": "com.saasure.db.dto.platform.entity.cvd.CVDType"
+  }
+}
+```
+
+### Update User Type
+
+<ApiOperation method="PUT" url="/api/v1/meta/types/user/{typeId}" />
+
+<ApiOperation method="POST" url="/api/v1/meta/types/user/{typeId}" />
+
+Updates an existing User Type. A PUT is a full replace operation; a POST is a partial update. The schema associated with this type is not editable with this API. If you want to edit the schema, use the [schema update API](/docs/reference/api/schemas/#add-property-to-user-profile-schema).
+
+##### Request Parameters
+
+The [User Type](#user-type-model) you wish to update
+
+##### Response parameters
+
+The updated [User types](#user-type-model)
+
+##### Request Example
+
+```bash
+curl -s -XPUT -H "Content-Type: application/json" -H "Authorization: SSWS ${api_token}" https://{yourOktaDomain}/api/v1/meta/types/user/otyfnly5cQjJT9PnR0g4 -d '{
+"description": "Updated description",
+"displayName": "Updated Name for UI",
+"name": "updatedTypeName"
+}'
+```
+
+##### Response Example
+
+```json
+{
+  "_links": {
+    "schema": {
+      "href": "https://{yourOktaDomain}/api/v1/meta/schemas/user/oscfnly5cQjJT9PnR0g4",
+      "method": "GET",
+      "rel": "schema"
+    },
+    "self": {
+      "href": "https://{yourOktaDomain}/api/v1/meta/types/user/otyfnly5cQjJT9PnR0g4",
+      "method": "GET",
+      "rel": "self"
+    }
+  },
+  "created": "2019-04-10T02:00:01.000Z",
+  "createdBy": "00ufnlhzppWItClAI0g4",
+  "default": false,
+  "description": "Any description that means something useful to you",
+  "displayName": "TheDisplayName",
+  "id": "otyfnly5cQjJT9PnR0g4",
+  "lastUpdated": "2019-04-10T02:00:01.000Z",
+  "lastUpdatedBy": "00ufnlhzppWItClAI0g4",
+  "name": "aNewType",
+  "ref": {
+    "id": "otyfnly5cQjJT9PnR0g4",
+    "refClass": "com.saasure.db.dto.platform.entity.cvd.CVDType"
+  }
+}
+```
+
+### Delete User Type
+
+<ApiOperation method="delete" url="/api/v1/meta/types/user/${typeId}" />
+
+Deletes a User Type permanently. This operation is not permitted for the default type, nor if there are non-deleted users with the specified type. After a User Type has been deleted, it cannot be used as the type for new users, and it no longer counts against the limit of 10 User Types.
```

Comment:
How about: "The delete operation is not permitted for the default user type, or for any user types type that has existing (non-deleted) users."